go.mod

```diff
@@ -226,6 +226,8 @@ require (
 	github.com/go-openapi/jsonreference v0.20.1 // indirect
 	github.com/go-openapi/swag v0.22.3 // indirect
 	github.com/go-task/slim-sprig v0.0.0-20210107165309-348f09dbbbc0 // indirect
+	github.com/gobuffalo/envy v1.7.0 // indirect
+	github.com/gobuffalo/packd v0.3.0 // indirect
```

Comment:
These don't look like needed for the PR. Could you remove them? Thank you!

Reply:
Done. Now there are somehow complaints about signoff and commit history is somehow botched. Should I maybe just re-introduce my changes to the current master and re-open the PR?

Reply:
Should be fine now